device-types/Minisforum/Minisforum-MS-01.yml

```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: Minisforum
+model: MS-01
+slug: minisforum-ms-01
+part_number: minisforum-ms-01
+u_height: 1
+is_full_depth: false  # 15.75"
+weight: 2.31
+weight_unit: lb
+front_image: true
+rear_image: true
+comments: '[Product Specs](https://store.minisforum.com/products/minisforum-ms-01?srsltid=AfmBOoowv0yNG0wjun4MV55hCnaCErDyzN48pzEYPOw1pgwQzolYaMET)'
+interfaces:
+  - name: ethernet-1/1
+    type: 10gbase-x-sfpp
+  - name: ethernet-1/2
+    type: 10gbase-x-sfpp
+  - name: ethernet-2/1
+    type: 2.5gbase-t
+  - name: ethernet-2/2
+    type: 2.5gbase-t
+rear-ports:
+  - name: usb-c-1/1
+    type: usb-c
+  - name: usb-c-1/2
+    type: usb-c
+  - name: usb-a-1/1
+    type: usb-a
+  - name: usb-a-1/2
+    type: usb-a
+module-bays:
+  - name: PS1
```

Comment:
As the power is integrated into the device it needs to be added as a `power-port` with `type: dc-terminal`